app/controllers/alchemy/admin/languages_controller.rb

```diff
@@ -1,11 +1,28 @@
 module Alchemy
   module Admin
     class LanguagesController < ResourcesController
+      def index
+        @query = Language.on_current_site.ransack(params[:q])
+        @languages = @query.result.page(params[:page] || 1).per(per_page_value_for_screen_size)
```

Comment:
The `|| 1` is unnecessary, if the argument for `page` is `nil`, it'll also show the first page. This is just FYI, no need to change it to being less explicit.